Character.h

```diff
@@ -53,22 +53,17 @@ struct Character
         std::cout << getName() << "'s attack damage level has been boosted to " << attackDamage << std::endl;
     }
 
-    void printStats()
-    {
-        std::cout << getName() << "'s stats: " << std::endl;
-        std::cout << getStats(); //make your getStats() use a function from the Utility.h
-        
-        std::cout << std::endl;
-        std::cout << std::endl;
-    }
+    void printStats();
+
 protected:
     std::vector<std::unique_ptr<Item>> defensiveItems;
     std::vector<std::unique_ptr<Item>> helpfulItems;
     int hitPoints, armor;
-    int attackDamage;
     bool isDefending = false;
 private:
+    int attackDamage;
     std::unique_ptr<int> initialHitPoints, initialArmorLevel, initialAttackDamage;
```

Comment:
`attackDamage` did not need to be moved from `protected` to `private`
- [ ] revise